build.gradle

```diff
@@ -101,9 +101,13 @@ dependencies {
 }
 
 task functionalTest(type: Test) {
-  description = "Runs Functional Tests"
+  description = "Runs end-to-end tests"
```

Comment:
perhaps it would be worth to rename the gradle task as well?

Reply:
Not a bad idea. Will do.